FAQ.md

```diff
@@ -111,11 +111,12 @@ We are a community initiative and you don't need to pay anything to stay on this
 * A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
 * And a live streaming service with multibitrate (if possible).
 
-After you have your m3u8 link, add your channel information onto the [Database](https://github.com/iptv-org/database) repository, and submit an [issue](https://github.com/iptv-org/iptv/issues/new/choose) to propose it to us.
+After you have your m3u8 link, add your channel information creating or editing the id (identifier) onto the [Database](https://github.com/iptv-org/database) repository. Then, submit an [issue](https://github.com/iptv-org/iptv/issues/new/choose) to propose it to us.
 
-Note: Because this is an community repository, we need some approvals from the administrators to continue. See [discussion](https://github.com/iptv-org/iptv/discussions/8412).
-
-Note 2: If your television station broadcasts from other television stations, please read the *[Can I safely add an FTA channel that broadcasts PPV events or pay programming?](#Can-i-safely-add-an-fta-channel-that-broadcasts-ppv-events-or-pay-programming)* section.
+Notes:
+* Because this is an community repository, we need some approvals from the administrators to continue. See [discussion](https://github.com/iptv-org/iptv/discussions/8412).
```

Comment:
```suggestion
* Because this is an community repository, we need the approval from two maintainers to accept your new additions. See [discussion](https://github.com/iptv-org/iptv/discussions/8412) for further explanations.
```